pages/dr9/sga.rst

```diff
@@ -10,178 +10,866 @@
 
 .. contents::
 
-*Page author and contact:* `John Moustakas`_
+*Primary contacts:* `John Moustakas (Siena College)`_ and `Dustin Lang (Perimeter Institute)`_.
 
-.. _`John Moustakas`: ../../contact/#other-experts
+.. _`John Moustakas (Siena College)`: ../../contact/#other-experts
+.. _`Dustin Lang (Perimeter Institute)`: ../../contact/#other-experts
 
-The Siena Galaxy Atlas (SGA) is an angular diameter-limited sample of galaxies constructed as part of the DESI Legacy Imaging Surveys. It provides
-custom, wide-area, optical and infrared mosaics (in :math:`grz` and :math:`W1-W4`), azimuthally averaged surface brightness profiles, and both aperture and integrated
-photometry for a sample of approximately 400,000 galaxies over :math:`20{,}000\,\mathrm{deg}^2`.
```

Comment:
This would work much better as `20,000 :math:...`

Reply:
I inherited the formatting of this particular bit of text from @geordie666's initial draft (see, e.g., https://github.com/legacysurvey/legacysurvey/blame/master/pages/dr9/sga.rst#L19), which propagated into my using math mode more than I should have.